ge/org.osate.ge.doc/docs/user_guide.md

```diff
@@ -247,6 +247,15 @@ If a a model element is renamed from the AADL text editor without using the *Ren
 2. Update the desired model references in the *New Reference* column.
 3. Select OK. The diagram elements will be restored.
 
+## Annotations and Documentation
+The following sections describe the graphical editors support for annotations and AADL documentation.
+
+### Annotations
+Diagrams can be annotated using the tools available in the *Annotation* section of the palette. The *Note* option allows add text notes to the diagram. Notes are part of the diagram and are not part of the AADL model. The *Note Reference* option can be used to associate a note with another element in the diagram.
```

Comment:
"The *Note* option allows add text notes..." should change to "... allows for adding text notes..." or something similar.